src/pages/index.js

```diff
@@ -1,58 +1,100 @@
 import { PopupWithForm } from "../components/PopupWithForm.js";
 import { PopupWithImage } from "../components/PopupWithImage.js";
+import { PopupWithConfirmation } from "../components/PopupWithConfirmation.js";
 import "../pages/index.css";
 
-// Render initial cards in using Section Class
+// API Initialized
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "91b601cc-d5b2-405d-a21a-614e7e8f57e7",
+    "Content-Type": "application/json"
+  }
+});
+
+// Render initial cards
 const section = new Section({
-  data: initialCards,
   renderer: (cardData) => {
     const cardElement = createCard(cardData);
     section.addItems(cardElement);
-  }
-}, '.galary__cards');
-section.renderItems(initialCards);
+    }
+  }, '.galary__cards');
+  
+
+const user = new UserInfo("#name", "#description", "#avatar");
+
+//Get User info
+api.getUserInfo()
+.then(userData => {
+  user.setUserInfo(
+    {name: userData.name,
+      about: userData.about,
+      avatar: userData.avatar});
+})
+.catch(err => {
+  console.error(`Error fetching user info: ${err}`);
+});
+
+api.getInitialCards()
+  .then(cards => {
+    section.renderInitialItems(cards);
+  })
+  .catch(err => {
+    console.error(`Error fetcting initial cards: ${err}`);
+  })
 
 
 /* Elements */ 
 
 //popups
 const editPopup = new PopupWithForm('#profile-edit-modal', handleProfileSubmit);
-const addcardPopup = new PopupWithForm('#add-card-modal', handleAddCardSubmit);
+const addCardPopup = new PopupWithForm('#add-card-modal', handleAddCardSubmit);
 const imagePopup = new PopupWithImage('#image-modal', handleImageClick); 
+const deletePopup = new PopupWithConfirmation('#delete-modal', handleDeleteSubmit);
+const avatarPopup = new PopupWithForm('#avatar-modal', handleAvatarSubmit);
 
 //button
 const profileEditButton = document.querySelector('#profile-edit-button');
 const addNewCardButton = document.querySelector("#profile-add-button");
+const avatar = document.querySelector("#avatar-overlay");
 
-//profile name & description / user info
+//profile name & description / user info / use Profile img
 const userNameInput = document.querySelector("#name-input");
 const userJobInput = document.querySelector("#description-input");
+const userAvatarInput = document.querySelector("#avatar-url-input");
 
-const userInfo = new UserInfo("#name", "#description")
 
 //forms
 const profileEditFormElement = document.forms["profile-edit-form"];
 const profileAddFormElement = document.forms["add-card-form"];
+const deleteFormElement = document.forms["delete-form"];
+const avatarFormElement = document.forms["avatar-form"]
 
 //validators
 const editFormValidator = new FormValidator(validationSettings, profileEditFormElement);
 const addFormValidator = new FormValidator(validationSettings, profileAddFormElement);
+const deleteFormValidator = new FormValidator(validationSettings, deleteFormElement);
```

Comment:
`deleteFormValidator` is not needed in the project because the confirmation modal doesn't have inputs